templates/alert-bar.php

```diff
@@ -17,7 +17,7 @@
 
 <div id="alert-bar-<?php echo $alert_bar->id; ?>" class="alert-bar<?php echo $class; ?>" data-slug="<?php echo $alert_bar->key; ?>-alert-bar" style="display: none;">
 	<?php if ( $alert_bar->countdown_end ) {
-		include_once 'alert-bar-countdown.php';
+		$this->template( 'alert-bar-countdown', array( 'alert_bar' => $alert_bar ) );
```

Comment:
Style thing: Our standard is now to use short array syntax.
```suggestion
		$this->template( 'alert-bar-countdown', [ 'alert_bar' => $alert_bar ] );
```

Reply:
Don't worry about this - keep the file consistent and we'll update them all in one go if/when we add coding standards.